themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+<!-- template for the subtitle-search component -->
+<tr class='subtitle-search-wrapper'>
+    <td colspan='9999' transition="expand">
+        <span v-if="loading" class="loading-message">{{loadingMessage}} <state-switch :theme="config.themeName" state="loading"></state-switch></span>
+        <div v-if="displayQuestion" class="search-question">
+            <div class="question">
+                <p>Do you want to manually pick subtitles or let us choose it for you?</p>
+            </div>
+            <div class="options">
+                <button type="button" class="btn-medusa btn-info" @click="autoSearch">Auto</button>
+                <button type="button" class="btn-medusa btn-success" @click="manualSearch">Manual</button>
+            </div>
+        </div>
+
+        <vue-good-table v-if="subtitles.length"
```

Comment:
```suggestion
        <vue-good-table v-if="subtitles.length > 0"
```
For clarity that `subtitles` is an array